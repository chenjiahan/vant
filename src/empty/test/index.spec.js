import Empty from '..';
import { mount } from '../../../test';

test('image slot', () => {
  const wrapper = mount(Empty, {
    scopedSlots: {
      image: () => 'Custom Image',
    },
  });

  expect(wrapper).toMatchSnapshot();
});

test('description slot', () => {
  const wrapper = mount(Empty, {
    scopedSlots: {
      description: () => 'Custom description',
    },
  });

  expect(wrapper).toMatchSnapshot();
});
