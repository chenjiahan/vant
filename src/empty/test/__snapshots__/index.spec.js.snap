// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description slot 1`] = `
<div class="van-empty">
  <div class="van-empty__image"><img src="https://img.yzcdn.cn/vant/empty-image-default.png"></div>
  <p class="van-empty__description">Custom description</p>
</div>
`;

exports[`image slot 1`] = `
<div class="van-empty">
  <div class="van-empty__image">Custom Image</div>
</div>
`;
